parlai/crowdsourcing/tasks/model_chat/analysis/compile_results.py

```diff
@@ -63,6 +63,13 @@ def setup_args(cls):
     def __init__(self, opt: Dict[str, Any]):
 
         super().__init__(opt)
+        # Validate problem buckets
```

Comment:
move out of `parlai/crowdsourcing/utils/analysis.py` here as it could be that "none_all_good" isn't necessary in `problem_buckets`.